tests/cli/test_cli.py

```diff
@@ -121,8 +159,12 @@ def test_query_info():
     assert "Current value" in result.stdout
 
 
-@pytest.mark.skip("Asking for password when it should not. Use local ganache endpoint")
-def test_query_parameters():
+@pytest.mark.skip("mocking ineffective")
```

Comment:
Change the reason to include that the test is broken, just so we know for sure in the future

Reply:
same with other skipped tests